src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under construction';
```

Comment:
- [ ] The inital state of your categories reducer should be an empty array. see tthe code snippet below as a guide
- code snippet
```javascript
 initialState: {
    categories: [],
  },
```

Reply:
Dear @Efiamotu-1 👋🏾
Thanks for the suggestion, and feedback,
 I appreciate it,
I already modified this part.
Best regards,